infrastructure/main.tf

```diff
@@ -21,6 +27,12 @@ locals {
   vault_name                = "${var.product}si-${var.env}"
 }
 
+data "azurerm_subnet" "postgres" {
+  name                 = "aks"
```

Comment:
probably not the best subnet,

you could either use one of the core-infra-subnet-x or create a new one

![image](https://user-images.githubusercontent.com/21194782/102078534-fbc85900-3e02-11eb-93e9-c2bc6192128e.png)

Reply:
I have used core-infra-subnet-x-env for for retrieving the corresponding subnet for each env .